src/commands/Util/serverinfo.ts

```diff
@@ -0,0 +1,42 @@
+import { AstraeaCommand, AstraeaCommandOptions } from '#lib/Structures/BaseCommand'
+import { Message, MessageEmbed } from 'discord.js'
+import { ApplyOptions } from '@sapphire/decorators'
+import dayjs from 'dayjs'
+
+@ApplyOptions<AstraeaCommandOptions>({
+  name: 'serverinfo',
+  aliases: ['si'],
+  description: 'Fetch the current guild\'s info'
+})
+export default class example extends AstraeaCommand {
+  public async run(message: Message): Promise<Message> {
+    const guild = message.guild
+    const roles: string[] = []
+
+    guild.roles.cache.forEach(r => {
+      if (r.name === '@everyone') return
+
+      return roles.push(r.name)
+    })
+
+    const embed = new MessageEmbed()
+      .setTitle(`${guild.name} | Guild Info`)
+      .setColor('BLURPLE')
+      .addFields(
+        { name: 'Guild ID', value: guild.id, inline: true },
```

Comment:
This could be more informative, most users won't profit from knowing an ID.
An @Mention is probably best

Reply:
experienced users / those who do need an id can just copy it from a mention